app/controllers/admin_controller.rb

```diff
@@ -143,6 +143,7 @@ def mark_comment_spam
         @comment.spam
         user = @comment.author
         user.ban
+        AdminMailer.notify_moderators_of_comment_spam(@comment, current_user).deliver_now
```

Comment:
```suggestion
```

We don't have to call it now.

Reply:
Do you prefer using `deliver_later`, maybe?

Reply:
Ok, I'm confused. Do yo refer to send the email now or call the mailer function in this method?

Reply:
I told you to remove this statement from here because as for now, the goal is just to create the mailers and not to activate them from anywhere and activate all the comment spam mailers together but I guess we can start this one, as this mailer seems to be ready. 

@jywarren what do you think? With this PR merged, whenever a moderator would mark the comment as spam, all the other moderators would be notified(Like, we do in node system).